### ISSUE TITLE
Disable submit button while submitting

### ISSUE THREAD

**Issue by Wgil** [ant-design/ant-design#27667]
- [X] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
I want to disable my form submit button without adding an additional state on every form.

### What does the proposed API look like?
Add `{isSubmitting: boolean}` property to `FormInstance` type which would toggled by ant form internal state on form submission.

**Comment by Wgil** [comment]
Can I work on this?

**Comment by kerm1it** [comment]
Yes，welcome to send PR.

**Comment by Wgil** [comment]
Looking at the `Form` component code, it seems that the functionality should be added to `rc-field-form` and then update the package here in ant design

**Comment by kerm1it** [comment]
Yes, you should add feature to `rc-field-form` and update package.json in antd.

**Comment by ayuechuan** [comment]
你可以自己在内部状态控制 提交表单button的状态(是否禁用)
